packages/calypso-e2e/src/lib/blocks/block-flows/instagram.ts

```diff
@@ -37,15 +37,17 @@ export class InstagramBlockFlow implements BlockFlow {
 	 * @param {EditorContext} context The current context for the editor at the point of test execution
 	 */
 	async configure( context: EditorContext ): Promise< void > {
-		const editorParent = await context.editorPage.getEditorParent();
-		const embedUrlLocator = editorParent.locator( selectors.embedUrlInput );
+		const editorCanvas = await context.editorPage.getEditorCanvas();
```

Comment:
It looks like we want to use `getEditorCanvas` for anything that renders inside the editor (because of the canvas iframe), right? This seems to be the case for simple and AT as well.

Reply:
It will probably be the main cause of failing tests until we update the parent selectors in the tests so that the frame context is correct (editor parent, aka Gutenframe vs. editor canvas). For elements that are located within the canvas, we should always be using the canvas frame as the root, and for the elements outside - the editor parent frame. It’s been the same with the upstream GB tests ever since the canvas iframe was introduced. Currently, in Calypso, not all selectors are scoped like that because part of the framework/tests was written before the iframe was introduced. Once we update those parts (like those that are failing now), it should stop failing regardless if the canvas iframe is there or not, thanks to https://github.com/Automattic/wp-calypso/pull/76085.